### DISCUSSION TITLE
Running LFRic Apps suite on Monsoon - SITE error

### DISCUSSION THREAD

**Discussion by ricky-lv426** [MetOffice/simulation-systems#266] [Monsoon]
Hi,

I'm trying to run a lfric_apps suite on Monsoon. These are the command I'm running:

```
export CYLC_VERSION=8
rose stem --group=lfric_atm_thai_hab1-C48_MG_xc40_intel_production-64bit-rbl32
cylc play vn1.1_add_ngarch_convection_app
```

The last command throws the error:

```
2024-10-28T11:32:59Z ERROR - Workflow shutting down - Jinja2Error: 'SITE' is undefined
  File /home/d05/eaolivier/cylc-run/vn1.1_add_ngarch_convection_app/run1/flow.cylc
   {# Contains some settings used for the workflow #}
   {% set site_vars = {} %}
   {% set kgo_vars = {} %}
   {% include "site/"~SITE~"/variables.cylc" %} <-- UndefinedError
```

I thought the SITE variable was automatically set on Monsoon. What am I doing wrong/missing?

I was just following the instructions here: https://metoffice.github.io/simulation-systems/WorkingPractices/TestSuites/lfric_apps.html

**Comment by james-bruten-mo** [comment]
Hi Ricky, Someone else had something similar on Monsoon recently. In that case they'd overwritten the SITE setting in their `~/.metomi/rose.conf` file. It doesn't look like you've done that, but maybe just having a `[rose-stem]` section is enough to overwrite that. Maybe try adding `automatic-options=SITE=meto` to that section.
